app/organisation/rest.py

```diff
@@ -284,6 +285,33 @@ def remove_letter_branding_from_organisation_pool(organisation_id, letter_brandi
     return {}, 204
 
 
+@organisation_blueprint.route("/notify-users-of-request-to-go-live/<uuid:service_id>", methods=["POST"])
+def notify_users_of_request_to_go_live(service_id):
+    template = dao_get_template_by_id(current_app.config["ORGANISATION_HAS_NEW_GO_LIVE_REQUEST_TEMPLATE_ID"])
+    service = dao_fetch_service_by_id(service_id)
+    organisation = service.organisation
+    notify_url = f"https://www.{current_app.config['NOTIFY_EMAIL_DOMAIN']}"
```

Comment:
`ADMIN_BASE_URL` can be used here instead of the email domain

Reply:
Changed in cad895b1